crates/ere-hosts/src/main.rs

```diff
@@ -185,6 +178,12 @@ fn main() -> Result<(), Box<dyn std::error::Error>> {
                 resource,
                 apply_patches,
             )?;
+            let config = RunConfig {
+                output_folder: cli.output_folder,
+                sub_folder: Some(el.as_ref().to_lowercase()),
```

Comment:
I would like to eventually reconsider this solution, but probably closer when we accept raw-ELFs potentially for externally compiled ELFs.